### PR TITLE
website/integrations: Mastodon integration

### PR DESCRIPTION
**Pull request by Tealk**
# Details
Extension of the instructions by the service Mastodon

### New Features
* Adds Mastodon instructions

### CODE REVIEW

File: website/integrations/services/mastodon/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Add the following
+
+```
+OIDC_ENABLED=true
+OIDC_DISPLAY_NAME=authentik
+OIDC_DISCOVERY=true
+OIDC_ISSUER=< OpenID Configuration Issuer>
+OIDC_AUTH_ENDPOINT=https://authentik.company/application/o/authorize/
+OIDC_SCOPE=openid,profile,email
+OIDC_UID_FIELD=preferred_username
```
Comment (BeryJu) [inline website/integrations/services/mastodon/index.md:51]: I don't know if this field is also used as a username in mastodon, if it isn't it makes more sense to use `sub` here as the username in authentik might change

**Review by BeryJu — CHANGES_REQUESTED** [review]

File: website/integrations/services/mastodon/index.md (hunk 1 above)
Comment (Tealk) [reply website/integrations/services/mastodon/index.md:51]: Can you explain in more detail what you mean by " to use `sub `here"?

Comment (BeryJu) [reply website/integrations/services/mastodon/index.md:51]: the `sub` field in the OIDC Token is a unique identifier that is (by default) generated based on the User ID and the hashed secret key, which unlike the user's username cannot change

with the current setting, if a user changes their username, they would get a new user in mastodon

File: website/integrations/services/mastodon/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: Mastodon
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Service Name
```
Comment (BeryJu) [inline website/integrations/services/mastodon/index.md:7]: replace placeholder

**Review by BeryJu — APPROVED** [review]

**Pull request merged**